### ISSUE TITLE
Setting Up a Multi-age Playroom: Displaying Learning Activities

### ISSUE THREAD

**Issue by Stephanymtr** [bmhco/shared#65]
Baby A (or A2) started crawling everywhere now, so A1’s learning activities (that were originally displayed at the bottom) have been moved up to the top of his toy shelving unit. Some of his activities have small pieces that are not safe to leave within reach of a crawling baby. A1 is quite tall now, so the activities at the top are not completely out of reach for him, but it’s almost out of sight. Naturally, he hasn’t been playing with those as much.

![image](https://github.com/bmhco/shared/assets/91985721/30c0df8f-9bcf-435c-9975-1e602d900b4a)

Displaying the toys on the higher shelves will limit the amount of learning activities A1 can have access to. As you can see on the photo above, there can only be 4 options max. 

Any ideas of what could be done to sort this out ?

**Comment by iteles** [comment]
Ideally we would have two of these shelves and use the bottom one for A1 and the top one for A2 (for now, as it won't be long before 👶 is pulling up to stand). 

But we just have to choose materials for the bottom shelves that aren't a safety hazard for 👶

**Comment by nelsonic** [comment]
@Stephanymtr thank you for opening this issue. Sadly, it’s addressing the “wrong problem”. We have too many learning materials and no catalog of them all. Most are stored in the attic in the storage I made for storing @iteles things to declutter the house. 

What we *need* is a priority on ***storage*** in what is *currently* the play/learning room  downstairs. And eventually in the outdoor area currently know as the “hole” but which will be the SunRoom. If I can just get back to Braga and have the time to focus on finishing it. ⏳😢

In the meantime we need a shelving unit where A2 (Nido) items can be placed. Preferably that creates a barrier that A2 cannot cross. Again needs to be prioritised. 
based on the priority, we can easily decide if we have time to *make* something custom or we need to *buy* something that fits *most* of the requirements. 

But IMO the biggest priority is making a decision on what items A1 should have in the play room. For example: the [mini] snooker table is nice, but the balls are the perfect choking hazard size for a curious toddler to asphyxiate on … 🎱☠️

Any learning material (or Toy) that could be dropped on the floor by a bigger child needs to be carefully considered. Not that we don’t want to have activities like beads/threading that an older kid can benefit from. Just that we need to have a well defined zone for that activity and never in the Nido prepared environment.

My view is that the next course of action is to capture the requirements for the Nido shelves and buy another KALLAX shelf system to serve as barrier+storage. 

@iteles please share your thoughts. 💭🙏

**Comment by iteles** [comment]
@nelsonic As I mentioned above, I agree that the immediate course of action is to get another KALLAX unit and also to ensure that the bottom shelves of both units are filled with items that are not choking hazards.

I agree that we also need to have a defined area for play with things that are unsafe for 👶 , especially sensory bases for example.

**Comment by Stephanymtr** [comment]
Alright! Thank you. 

Given the immediate course of action here is to purchase another piece of furniture, I will keep A1's toys where they are for now.

Is it okay if I move A2's toys to the bottom of the current shelving unit ?

**Comment by nelsonic** [comment]
@Stephanymtr if you don’t mind using our Van (the one Miguel drives) to head to IKEA I can place a click-and-collect order for the shelving unit. @iteles is this OK?

**Comment by iteles** [comment]
@nelsonic Apologies, a 👍  is not very clear. 

Seeing as I have been unsuccessful in sourcing one for the past 4 weeks second hand, I agree it's time to get this moving on this!

**Comment by Stephanymtr** [comment]
I'm going to be at the house today until 4:30pm (cutting my lunch brake in half). Please let me know if/when the order is ready and what the instructions are.

**Comment by nelsonic** [comment]
Did you agree on what dimensions of [`KALLAX`](https://www.ikea.com/pt/pt/search/?q=kallax) we need for the space available? 
Are we just going to buy the _same_ `77x147 cm` (4x2 spaces) one we already have? 
e.g: https://www.ikea.com/pt/pt/p/kallax-estante-ef-carvalho-c-velatura-branca-00324518/
![image](https://github.com/bmhco/shared/assets/194400/fea2be87-5919-4bb1-95b0-6c2998e7f4e0)

***OR*** are we thinking of getting the taller one `112x147 cm` (4x3 cubes):
https://www.ikea.com/pt/pt/p/kallax-estante-ef-carvalho-c-velatura-branca-40409935/#content
![image](https://github.com/bmhco/shared/assets/194400/1ecc07b1-1aac-4523-aeba-c0d076f8cc22)

which on its' side is `112cm` tall and thus just above the head height of the average 2-year old and thus creates a visual barrier in the room. 💭 

The _reason_ I'm axsking this is simple: for the same floor area use we get 50% more storage. 🙌

**Comment by iteles** [comment]
100% the same size we already have. The visual and light barrier wouldn't be great and also not very Montessori to have a bunch of toys in view but inaccessible to kids. 

If we want one for storage we should get that separately but I don't think these are a good storage solution.

**Comment by nelsonic** [comment]
@Stephanymtr I've placed the order: **`1374714556`**
![IKEA-order-Kallax](https://github.com/bmhco/shared/assets/194400/118f0bbb-03a8-427b-bff2-1ad2b0a79c70)

Will forward the order to you by email so that you know what to collect. 
Do you have someone available who can help with collection?
Or should we send Miguel with you? 💭

**Comment by nelsonic** [comment]
@iteles if you visit https://www.ikea.com/pt/pt/purchases/ you'll see the order is "Em Curso"
![image](https://github.com/bmhco/shared/assets/194400/30ada302-f859-4f23-ae9e-3f3daf8b02f6)

But if you try to click/tap on the order to view details it errors ... 
![image](https://github.com/bmhco/shared/assets/194400/d8615c54-f3dd-498d-9dd8-2c2a4a190b28)

So much for the amazing e-commerce website and e-POS integration they spent 100M+ on ... 🙄 
If you can access it or the email confirmation comes through, please forward to @Stephanymtr thanks. 👍

**Comment by Stephanymtr** [comment]
No, I don't know anybody available to help me carry this shelving unit.

Your total was above 65€... How many of those did you buy? 

I also haven't received any emails. Will the screenshot above suffice or shall we wait for a confirmation email?

**Comment by nelsonic** [comment]
Hi @Stephanymtr, yes. Slightly more than the `€65`, yes. 💸 🙄 
But hopefully we can get a bit more storage without _me_ being the bottleneck to build/make it. 🤞 
I ordered _three_ of the shelving units and a couple of smaller items. 
One includes a Desk 
https://www.ikea.com/pt/en/p/kallax-linnmon-desk-combination-white-white-stained-oak-effect-s89481699/#content
![image](https://github.com/bmhco/shared/assets/194400/1f4ad55e-60d5-466c-832d-83f8c582a1b1)

I've forwarded the order email to you. ✉️

**Comment by Stephanymtr** [comment]
Done! Following the emails instructions not to bring Miguel to Ikea with me. I had to load these heavy four boxes by myself (+ other small products). Ikea employees can't leave the store and there wasn't anybody outside (35C 🔥) to ask to help me . Miguel helped unload everything off the car to the hallway near the playroom.

 ![image](https://github.com/bmhco/shared/assets/91985721/980b72f8-b7ae-43c8-836a-3c7a22b5c38c)

![image](https://github.com/bmhco/shared/assets/91985721/fefab682-b6c7-4ba9-94a6-a5bea18f3011)

**Comment by nelsonic** [comment]
Wow! What a workout! 🏋️‍♀️ 📦 
Thank you for doing that. 🙏
